google/cloud/bigtable/rpc_retry_policy.h

```diff
@@ -24,6 +25,25 @@ namespace google {
 namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
+/// An adapter to use `grpc::Status` with the `google::cloud::*Policies`.
+struct SafeGrpcRetry {
+  // Sometimes we need to use google::protobuf::rpc::Status, in which case this
+  // is a easier function
+  static inline bool IsTransientFailure(grpc::StatusCode code) {
+    return code == grpc::StatusCode::OK or code == grpc::StatusCode::ABORTED or
+           code == grpc::StatusCode::UNAVAILABLE or
+           code == grpc::StatusCode::DEADLINE_EXCEEDED;
+  }
+
+  static inline bool IsOkay(grpc::Status const& status) { return status.ok(); }
```

Comment:
Nit: I generally see these methods that check around 200/"OK"  logic spelled as "Ok", rather than "Okay".

Reply:
Fixed.